### PR TITLE
update sponsorship instructions

### PR DESCRIPTION
**Pull request by jodygarnett**
This PR addresses feedback received on:

* confusing instructions on when to email treasurer or expect an email from treasurer
* adds screen snaps from PayPal and GitHub Sponsors
* collects the recommendations and recognition text from the wiki, along with the point that sponsorship is not an alternative to commercial support
* includes all OSGeo sponsorship tiers (as large organizations and countries using GeoServer can afford to support the project financially)

### CODE REVIEW

File: sponsor/index.html [hunk 1]
```diff
@@ -102,8 +105,14 @@ <h4>Sponsorship via Invoice</h4>
       treasurer@osgeo.org
     </a> to express your interest.
       </li>
-      <li>For inclusion on the OSGeo <a href="https://www.osgeo.org/sponsors/">sponsorship</a> page also attach your company logo.</li>
-      <li>Example email sponsoring GeoServer:
+      <li>We need some information to properly thank you for your support:
```
Comment (aaime) [inline sponsor/index.html:108]: I'd tweak the first sentence at the beginning of the "Sponsorship via Invoice" to something like:

"Great for companies and public institutions seeking to perform a one-time donation"

Or something like that

File: sponsor/index.html [hunk 2]
```diff
@@ -123,75 +132,242 @@ <h4>Sponsorship via Invoice</h4>
...
-        </blockquote>
-      </li>
     </ol>
+    
 </section>
+
+<section id="recommendations">
+    <h2>Sponsorship Recognition and Recommendations</h2>
+    <p class="lead">
+      Financial support indicates a commitment to the sustainability of the GeoServer, for those that depend on GeoServer's availability. Sponsorship is not a replacement for commercial support.
```
Comment (aaime) [inline sponsor/index.html:204]:
```suggestion
      Financial support indicates a commitment to the sustainability of the GeoServer, for those that depend on GeoServer's availability.
```

"Sponsorship is not a replacement for commercial support" needs its own place IMHO, explaining that it's not directed to a specific goal, unlike commercial support

File: sponsor/index.html [hunk 3]
```diff
@@ -123,75 +132,242 @@ <h4>Sponsorship via Invoice</h4>
...
+    <h4>Individuals</h4>
+    <p>
+       Donations help the project steering committee support developers in small ways.
+       Examples include covering travel expenses and code-sprint attendance.
+    </p>
+    <ul>
+      <li>$5/monthly Enthusiast:Fans of OSGeo, badge your repository as supporting OSGeo with a small donation.</li>
+      <li>$10/monthly Supporter: GIS Professionals, name listed as 2024 Donations on GeoServer website.</li>
+      <li>$50/monthly Bronze Sponsor: Individual consultants, Logo listed as 2024 Sponsor on <a href="https://www.osgeo.org/sponsors/">GeoServer website</a>, cross-listed to OSGeo website as a bronze sponsors, bronze sponsor logo, be sure to setup your <a href="http://osgeo.org/service-providers/">service provider page</a>.</li>
+    </ul>
```
Comment (aaime) [inline sponsor/index.html:218]: You go on talking about OSGeo, but most GeoServer users I know have no idea what OSGeo is to start with.
I'd add a statement clarifying the relationship and indicate that with a big enough donation, one gets recognized also as an OSGeo sponsor, as a bonus.

Comment (jodygarnett) [reply sponsor/index.html:218]: Except sometimes we do fundraising for a specific goal; the point is they do not get to choose the goal :)

File: sponsor/index.html (hunk 2 above)
Comment (jodygarnett) [reply sponsor/index.html:204]: Except sometimes we do fundraising for a specific goal; the point is they do not get to choose the goal :)

If I start the sentence with that statement would it be visible enough?

Comment (aaime) [reply sponsor/index.html:204]: Yes

File: sponsor/index.html (hunk 1 above)
Comment (jodygarnett) [reply sponsor/index.html:108]: "public institutions" good phrasing

File: sponsor/index.html (hunk 3 above)
Comment (jodygarnett) [reply sponsor/index.html:218]: I thought I did that at the top of the page, let me try and address it here also.

Comment (jodygarnett) [reply sponsor/index.html:218]: image

**Pull request merged**